.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -12,36 +12,110 @@ on:
+  cancel-in-progress: true
+
 jobs:
   set_wheel_build_matrix:
     name: "Set wheel build matrix"
     uses: ./.github/workflows/set_wheel_build_matrix.yml
     with:
       event_name: ${{ github.event_name }}
 
-  linux-wheels-aarch64:
+  build_dependencies:
     needs: [set_wheel_build_matrix]
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        arch: [aarch64]
+        exec_model: ${{ fromJson(needs.set_wheel_build_matrix.outputs.exec_model) }}
+        kokkos_version: ${{ fromJson(needs.set_wheel_build_matrix.outputs.kokkos_version) }}
+        container_img: ["quay.io/pypa/manylinux2014_aarch64"]
+
+    name: Kokkos core (${{ matrix.exec_model }}::${{ matrix.arch }})
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Cache installation directories
+        id: kokkos-cache
+        uses: actions/cache@v3
+        with:
+          path: ${{ github.workspace }}/Kokkos_install/${{ matrix.exec_model }}
+          key: ${{ matrix.container_img }}-kokkos${{ matrix.kokkos_version }}-${{ matrix.exec_model }}
+
+      - name: Clone Kokkos libs
+        if: steps.kokkos-cache.outputs.cache-hit != 'true'
+        run: |
+          git clone https://github.com/kokkos/kokkos.git
+          cd kokkos
+          git checkout ${{ matrix.kokkos_version }}
+          cd -
+          pushd . &> /dev/null
+
+      - uses: docker/setup-qemu-action@v2
+        name: Set up QEMU
+
+      - name: Build Kokkos core library
+        if: steps.kokkos-cache.outputs.cache-hit != 'true'
+        run: |
+          mkdir -p ${{ github.workspace }}/Kokkos_install/${{ matrix.exec_model }}
+          cd kokkos
+          docker run --platform linux/aarch64 \
+            -v /var/run/docker.sock:/var/run/docker.sock \
+            -v `pwd`:/io \
+            -v ${{ github.workspace }}/Kokkos_install/${{ matrix.exec_model }}:/install \
+            -i ${{ matrix.container_img }} \
+            bash -c "git config --global --add safe.directory /io && \
+            cd /io && \
+            python3.9 -m pip install ninja && \
+            ln -s /opt/python/cp39-cp39/bin/ninja /usr/bin/ninja && \
+            cmake -BBuild . -DCMAKE_INSTALL_PREFIX=/install \
+                            -DKokkos_ENABLE_COMPLEX_ALIGN=OFF \
+                            -DKokkos_ENABLE_SERIAL=ON \
+                            -DKokkos_ENABLE_${{ matrix.exec_model }}=ON \
+                            -DKokkos_ENABLE_DEPRECATION_WARNINGS=OFF \
+                            -DCMAKE_CXX_STANDARD=20 \
+                            -DCMAKE_POSITION_INDEPENDENT_CODE=ON \
+                            -G Ninja && \
+            cmake --build ./Build --verbose && \
+            cmake --install ./Build; "
+          cd -
+
+  linux-wheels-aarch64:
+    needs: [set_wheel_build_matrix, build_dependencies]
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
         arch: [aarch64]
+        pl_backend: ["lightning_qubit"]
```

Comment:
Don't we want to build `lightning_kokkos` too, here and a few other places (wheel_*)?